core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -45,7 +45,9 @@ import { Subscription } from 'rxjs';
 
 // TODO(#9186): Change variable name to 'constants' once this file
 // is migrated to Angular.
-import storyNodeConstants from 'assets/constants';
+import { AppConstants } from 'app.constants';
+
+const storyNodeConstants = AppConstants.commonConstants;
```

Comment:
Again, use `AppConstants` directly.

Reply:
ditto elsewhere, also you can probably remove the comment starting with `TODO(#9186):`

Reply:
Hi, I addressed all comments, can you take a look?

Reply:
> ditto elsewhere, also you can probably remove the comment starting with `TODO(#9186):`
All comments addressed